README.md

```diff
@@ -203,8 +212,7 @@ Out of the 18 identifier types mentioned in HIPAA Safe Harbor method (2)(i), thi
 
 This configuration file is provided in a best-effort manner. We **strongly** recommend that you review the HIPAA guidelines as well as the implementation of this project before using it for you anonymization requirements. 
 
-The safe harbor configuration file can be accessed [here](src/Fhir.Anonymizer.Tool/configuration-sample.json).
-
+The safe harbor configuration sample for [R4](src/Fhir.Anonymizer.R4.CommandLineTool/configuration-sample.json) and [Stu3](src/Fhir.Anonymizer.Stu3.CommandLineTool/configuration-sample.json) are given in repo.

```

Comment:
- configuration samples. plural.
-  Stu3 => STU 3
-  The safe harbor configuration files can be accessed via [R4](src/Fhir.Anonymizer.R4.CommandLineTool/configuration-sample.json) and [STU 3](src/Fhir.Anonymizer.Stu3.CommandLineTool/configuration-sample.json) links?